specifyweb/specify/models.py

```diff
@@ -1442,6 +1442,8 @@ class Collectionobject(models.Model):
     yesno4 = models.BooleanField(blank=True, null=True, unique=False, db_column='YesNo4', db_index=False)
     yesno5 = models.BooleanField(blank=True, null=True, unique=False, db_column='YesNo5', db_index=False)
     yesno6 = models.BooleanField(blank=True, null=True, unique=False, db_column='YesNo6', db_index=False)
+    ismemberofcog = models.BooleanField(blank=True, null=True, unique=False, db_column='IsMemberOfCog', default=False, db_index=False)
```

Comment:
Hmm is this needed here? Instead of a physical field, what if we have a virtual field (see isonloan for preparation). It'll just check if that entry exists in join table.

Reply:
The geo-analyst mentioned that this field might be needed.  I just put it in here for now.  Making it a virtual field sounds good to me.

Reply:
Careful that the virtual field isn't queriable. But, you can make that query pretty easily